R/make-test.R

```diff
@@ -27,25 +27,11 @@ make_test <- function(elts, opt = demo_options(),
   shiny::shinyApp(
     ui = ui(opt = opt),
     server = server(elts = elts, opt = opt,
-                    custom_admin_panel = custom_admin_panel),
-    onStart = setup_on_start(opt)
+                    custom_admin_panel = custom_admin_panel)
```

Comment:
How come we are no longer using the `onStart` argument? I worry that this will introduce subtly different behaviour.

Reply:
Hi Peter,

I believe I did this so that the session/state objects can be accessed. Seems like it's mentioned in one of my commits.

Reply:
Makes sense! OK, this is non-critical code anyway, I don't think most people use that argument, so if it works for you that's good enough for me.